kura/org.eclipse.kura.core/src/main/java/org/eclipse/kura/core/data/DataServiceImpl.java

```diff
@@ -526,6 +536,27 @@ public void run() {
         return autoConnect;
     }
 
+    private void getThrottle() {
+        if (this.properties.get(PUBLISH_RATE) != null && this.properties.get(BURST_LENGTH) != null) {
```

Comment:
Isn't it more readable if we save `this.properties.get(PUBLISH_RATE)` and `this.properties.get(BURST_LENGTH)` into local variables?